webpack.config.js

```diff
@@ -0,0 +1,233 @@
+const path = require('path');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const PeerDepsExternalsPlugin = require('peer-deps-externals-webpack-plugin');
+const postcssImport = require('postcss-import');
+const postcssPresetEnv = require('postcss-preset-env');
+const postcssReporter = require('postcss-reporter');
+const postcssCustomProperties = require('postcss-custom-properties');
+const postcssCustomMediaQueries = require('postcss-custom-media');
+const postcssPostcssColorModFunction = require('postcss-color-mod-function');
+
+const browserslist = {
+  development: ['chrome', 'firefox'].map(
+    browser => `last 2 ${browser} versions`
+  ),
+  production: ['>1%', 'not op_mini all', 'ie 11'],
+};
+
+const sourceFolders = [
+  path.resolve(__dirname),
+  path.resolve(__dirname, '../examples'),
+  path.resolve(__dirname, '../src'),
+];
+
+module.exports = {
+  entry: {
+    main: './src/index.js',
+    utils: './src/utils/index.js',
+  },
+  mode: 'production',
+  output: {
+    path: path.join(__dirname, 'dist'),
+    library: 'McUiKit',
+    libraryTarget: 'umd',
```

Comment:
@montezume asked (moving the discussion here for threaded conversation):

> Was wondering if you considered also creating an ES bundle as well?

We could do that, but I am not sure how it would work with CSS modules. I guess we would still need to run webpack, but we just would transpile less?